crates/neon-runtime/src/napi/error.rs

```diff
@@ -1,11 +1,24 @@
-use raw::Local;
+use raw::{Env, Local};
 
-pub unsafe extern "C" fn throw(_val: Local) { unimplemented!() }
+use nodejs_sys as napi;
 
-pub unsafe extern "C" fn new_error(_out: &mut Local, _msg: Local) { unimplemented!() }
+pub unsafe extern "C" fn throw(env:Env, error: Local)->bool {
+    let status=napi::napi_throw(env,error);
+    assert_eq!(status, napi::napi_status::napi_ok);
+    status==napi::napi_status::napi_ok
+}
 
-pub unsafe extern "C" fn new_type_error(_out: &mut Local, _msg: Local) { unimplemented!() }
+pub unsafe extern "C" fn new_error(out: &mut Local, env:Env, code:Local, msg:Local)->bool {
+    let status=napi::napi_create_error(env,code,msg,out);
```

Comment:
Like `throw` above, all these functions (`new_error`, `new_type_error`, `new_range_error`) should also be converted to assert and not return `bool`.